docs/components/modeler/bpmn/link-events/link-events.md

```diff
@@ -0,0 +1,47 @@
+---
+id: link-events
+title: "Link events"
+description: "Link events are pairs of intermediate events, one throwing and one catching, that link the two together."
+---
+
+Link events are pairs of intermediate events, one throwing and one catching, that link the two together.
```

Comment:
:wrench: This is not 100% correct. :smile: 

A process may contain multiple throwing link events that link to one catch event.